conf/test.config

```diff
@@ -26,7 +26,7 @@ params {
     skip_krona                    = true
     min_length_unbinned_contigs   = 1
     max_unbinned_contigs          = 2
-    busco_reference               = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
+    //busco_reference               = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
```

Comment:
Any reason why this is commented? 
The param below (busco_clean) is related...

Reply:
Whoops! Had that commented out for testing as it conflicts with --skip_binqc, but didn't mean to actually commit the change...